tock/tock/static/sass/core.scss

```diff
@@ -28,6 +28,9 @@ li {
 }
 .usa-form select {
   @extend .usa-select;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
```

Comment:
For some reason, the `select` element wasn't picking up the fact that it should hide the default browser chrome 🤷

Reply:
Might be worth pinging @dano about. I'd think .usa-select would, but he may have thoughts.